roles/kubernetes/preinstall/templates/resolved.conf.j2

```diff
@@ -1,10 +1,6 @@
 [Resolve]
-{% if dns_late %}
```

Comment:
I see that `dns_late` is used only here and in a couple of `when` statements, would we be able to get rid of that var (or rename it to something more explicit) any time soon?

Reply:
We can't get rid of dns_late. It's used when resolvconf_mode is set to host. If you set it up too early to point to coredns, then all your dns lookups have to timeout. Since ssh is one of those services that uses dns lookups, it dramatically slows down deployment. We only enable setting coredns in /etc/resolv.conf it in the late phase on purpose.